templates/clamd.conf.j2

```diff
@@ -0,0 +1,9 @@
+# {{ ansible_managed }}
+
+LocalSocket              /run/clamav/clamd.sock
+PidFile                  /run/clamav/clamd.pid
+TemporaryDirectory       /tmp
+LogFile                  /var/log/clamav/clamd.log
+LogSyslog                true
+User                     clamav
```

Comment:
Is the user created automatically?
Also, these values should be configurable.

Reply:
Please don't make me to. It's one of the few things **most** distributions agree about.

Reply:
They even agree about the name of the user (I think even Debian which is the distribution where **everything** has a different name)